extensions/Starry/SimpleEncrypt.js

```diff
@@ -0,0 +1,508 @@
wQHlIf6y3yf2SNHwrqCI5FM5CioJ9Mu1F5ALthuQkSk07Qtj0Tm3Ik2xjtWThkxqukbKa+o3TLZWw6kaOyRzWQA5Yr+aiOYnLMnyd6H9TWOm4c2dv/Z3KLtM46b8KA5T1y9lCYcN8MtunWIEffiqQAODNrPgKpaPAbbz6my99OkXXHfVOLWskugvYbsvYDjRHuvwX8Z7XUqU6RQBbJnAeYPNL6CUxbCU5xzNFSrJtbeo0wnBh9IwczgLhmLbzxyQTxlrUeBKeJXO7O+wfHh8KMSF9XwAtzoxIOlfF1OwajhPEIIQ8VuNFcOeASzpQbWkAimMke4xjQB+1th4gC5ut4c+VR9lfLenGjFukBqCHEedM4YaHLWORDkesBjI2WivsD8aiNAxfOwvs78wBGKmNxzmwGU2C7cz1zMllR9u4s0mCPvqQ0Az0um/togtUzrcp0tUg44wvjIGCnj9EUiMeV3IoEJJke/2KdVkbKE3OZcEaBy/UX2V2TMMeuhNi+aHPCIyFM5L5E577FMWK+91+CT7vh58MnLu4l+vKVFP9rQoh/POG6UyJdoTv843brKFCRu03uHW3TRQqAmgRaPsXapqyrnFFLVBgEYPV68oKi9khVPxeIEBDcKzv/S/Ag1UNjDrmKcyvlsEMfZ+20jTGMnHS2IG6cZPhHxVC58EstUDtj7IMPFwsvJ+EaJx2LbK2FFi2issGcIboy017lwkfid+RjvU4NPPIwad9cG7aPgERLfUJNDIhuWS0/bru35qjAvtP10T/seec6zHM1Ysj1/U6UvMj8RNVsqy+4UyAFnI3PA/AQjc7Hj61mP7PqT5geJULgudYuyL0YC3TVrU/0ztus6yG0j5KtdpI4ANaBv7ZMGI3verYAx1hwoF+WLQwcSYQDXhVZfM9TmF4zkxEZimO9WtR48EZGe6ovMjzXwWrtZmYCI3PHa/g7ar9R2OXsfZX9BXFS0VxTQ046zgE/W7Z5w2Px4E9FDm1q044i9f2rG86357PBcorvWTKRJ3TE0TgvOAOy1O1Ra/OjFfwn4scLs8tghFk8B/Y3FJ2I/G2jXhUlaveH9XYHrawaKwUWi/c1h7z37lLa/MG5U4M5G8M00wCdGBXh8uw9qGZ6lHYzsb2mXLOzfGD7JgcMix8nuZ2deEFyU4E/oAT7BGDUc3dc2VmIOuLubdtF2MPIpHIJC7dUiXUhEz/Kg1WjsYE6Zj/4era8iwsDmCiORveJjACI3bI5ysWkA8lqmiDryFBvJiZVTzI94GxKSozwdxldgmDSaix3LbEnNPeecha7xSM3hFuTFnHOhxk2sXCxTpLRxDMiEqv4jkZ/USNlpkosdjMgFagAhdtfrxADtYE0elKvhDFAultnC4YQCdCHjlBMXReKp9bsmHDcPGcfN5jbtOPLWdMDE9vrmlUR3n92iu4aJLl/M7V9ADRSIkQDWltPguGmFpzg7FourTvHafWomLIL/gPEV283IZEWYS1JtnVMZnwTtL4PT1euhcKTE4inLKYrYsRpDRYPHMjCGy81b2a4tF9XPyHGyjU2k3RZvLWT8BRajJlPuuBf8MKaU6yQHfnxWWBvZ4DzZan25jwiECTn05TY5cXw45gTnPPPNT0R/K55lIULARqp6qK8bCDD/n3V8PePMtmtHnAA5MTIREwGy58mRFyMFoFzNcw9Gjrwef7Bd76fJRKLFeRHmRxtJmU454OL4ZIqUcU7K4JZpG22EScPMDzSfnJzk7PApniGCFDtOUMTI1TtB35rdDk6m9UAmnKHBCTXnS4K+mnZj7X0Fpyhq8owTvbC7RQ9saNF9G6jjwDl8coZxg872yjnjdNeadodx896hFi2chdhBwQ6J+CaHXADHsXapB3gstt0cOKMXuAiyKyC+gexv9yEYN0pBbhdfc7hGkM+m73TCU0ywOrhpZMAnlX3eNy9N4BMNnrLah9a9hDOU7Zb2UymX095HMW8RBg8zTgKO6zpqwMhVCVLA57lOViZbek8xOfamAbarzTFmmRvc5hiIEHgjexHMi9pia6imTplTGNHfSu0CIWIkjq8y1zfmNgdtDjfM5InJbW6AGcPmyBfGJ5bxYX1/pcc9NkKmqaHSRLtCznrsbQ4iU8TJARcjSEJ/YyMxscyY0iZLuca59LX29VR91bnGzPyoIjKR66m0zaAdy5mL7ab7BZlOEXZXYtLAkciIdgt7b7DzWHd9RtlPz34eYsTWcBFi77mTALguKvhGtiePbG7RfRuJ7t/Qoie2zThtgucv64EbVhDdPdyiu9a06OolACOmUjsjB57y2E01PgEYG+WhUYnHkFslIcZRjnY9zNvKvHC4SIPjAGYn1F+QedFkjRp4HU11jRoNnpLmx3Y6+M5TGZlHEJ6SmNEafVPxlMfuInbbxjVqnGHvyKD9y4mLOvsXknGTwByy8UmHDIDMazXDx3/rU+VwIZm31NwuT26/1GThaTOLouh4rOcL7qelkO3p6/yxIrJRWRSuS9XT6eyROYW+0kfom6OKvtaiFJ/nxtde1QLIjI3guMPvfkSlSp4IQzH15t8sY0SaT4k5lHl+uPHJpq8wv2KETLEuavPiWRf2sJX7gmKctfqWBXcZUO7dv6x1DM+LG8rQtpcqX7SHRArchRXob22cPEwPN0AJjS+ir/siW19EntFX3Be49pjDg2T/KvuWlhkTsNdeBpv9vXY/rpodbBInJOCMbPZTY68lvBCYXE7fDPjmwAmiBze26P6u48bcMDXzw0Zg2Wyi9w2b+jZtunOIaNFsLi221/besx5d+BCLx1i7q2Sylk6LVHsPMJC8ODAwTpV+CvYAYsKCTBqvPQPlRfunsfeAvQ7i3AAeC46vY//c6QrKC/gEsme9xlOR9r6jZiyecs7ZKjwVo28unGE7b7C9ukwrKtctguMS7HU5LzHjJDmJJnXCatR4mTGgByw291wROZosGOTWmmE+Ql+EjHPaVPa8QH9j9c3SbrFIAMYG6/FXyHHpY
```

Comment:
Regarding the readability of variable names: it is not recommended to use a mixture of Hanyu Pinyin initials + English semantics. (Recommendations only)